build/gulp/tasks/test-projects/typings/index.tsx

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
here we might want to simplify things, just to eliminate any maintainability burden. Essentially, what we would like to check is that all the types of stardust module are consistent and have proper imports (i.e. absense of imports like `import SomeType from 'src/..'`). For that the following basic code would be just enough:

```js
import * as stardust from '@stardust-ui/react'
const Test = () => console.log(stardust)

export default Test
```

Current version of code doesn't introduce any additional problem-capturing capabilities, as I see it - am I missing something?